Source/MQTTnet/Implementations/MqttWebSocketChannel.cs

```diff
@@ -109,6 +110,38 @@ public async Task WriteAsync(ArraySegment<byte> buffer, bool isEndOfPacket, Canc
             await _webSocket.SendAsync(buffer, WebSocketMessageType.Binary, isEndOfPacket, cancellationToken).ConfigureAwait(false);
         }
 
+        public async Task WriteAsync(ReadOnlySequence<byte> buffer, bool isEndOfPacket, CancellationToken cancellationToken)
+        {
+            cancellationToken.ThrowIfCancellationRequested();
+
+#if NET5_0_OR_GREATER
```

Comment:
We don't need the NET5_0_OR_GREATER code because the new version will support .NET 8.0+